File: CS50/cash.c
#include <stdio.h>
int coin(int bucks)
{
	int r=0,q=0,d=0,x=0,p=0;
	if (bucks>=25)
	{
		q=bucks/25;
		r=bucks%25;
		
	}
	if (r>=10)
	{
		d=r/10;
		r=r%10;
		
	}
	if (r>=5)
	{
		x=r/5;
		r=r%5;
		
	}
	if (r>=1)
	{
		p=r/1;
	}	
		
		

	return (p+q+d+x);
}
int main ()
{
	float doller=0;
	while (doller<=0)
		
	{	
		
		printf("Cash : ");
		scanf("%f\n", &doller);		
	}
	float b=doller*100;
	int bucks=(int)b%100;
	printf("coin : %d\n",coin(bucks));
	return 0;	
}

File: CS50/week6/cash.py
def main():
	total_coin = 0
	doller = 0
	print ("hello cash")
	while doller <= 0:
		doller = float (input ("Cash : "))
	tmp = doller *100
	bucks = int(tmp % 100)
	coin = [25, 10, 5, 1]
	for i in range(4):
		coins=coin_count(bucks,coin[i])
		total_coin = total_coin + coins
		bucks = bucks - coins*coin[i]
	print (f"coin : {total_coin}")
	
	
def coin_count(bucks,value):
	coinlocal = 0
	if(bucks>=value):
		coinlocal = int(bucks / value)
	return coinlocal
		
main()
